Dockerfile

```diff
@@ -1,4 +1,6 @@
 FROM python:3
 
+RUN pip install pep8
```

Comment:
Please use a `requirements.txt` file to store your dependencies with a fixed version and install them with the `make install` command.